common/install_cuda_aarch64.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+
+set -ex
+
+function install_124 {
+  echo "Installing CUDA 12.4 and cuDNN 8.9 and NCCL 2.19.3 and cuSparseLt-0.5.2"
+  rm -rf /usr/local/cuda-12.4 /usr/local/cuda
+  # install CUDA 12.4.0 in the same container
+  wget -q https://developer.download.nvidia.com/compute/cuda/12.4.0/local_installers/cuda_12.4.0_550.54.14_linux_sbsa.run
```

Comment:
should add a local variable to ensure the correct version is propagated throughout the function
```suggestion
  local cuda_version='12.4.0_550.54.14`
  wget -q "https://developer.download.nvidia.com/compute/cuda/12.4.0/local_installers/cuda_${cuda_version}_linux_sbsa.run"
```

Same for the other places below, and similarly for CUDNN

Reply:
Thanks for the suggestion, that's a good point. Planning to refactor as the next step, but for this PR targeting to make the wheel work.